checksum/example_test.go

```diff
@@ -28,6 +28,7 @@ func ExampleChecksum_marshalling() {
 		cs    Checksum
 	)
 
+	//nolint:staticcheck
 	rand.Read(csRaw[:])
```

Comment:
Why not use `rand.Read` from the `crypto/rand` package as the linter suggests? Perhaps it's better to avoid deprecated functions, even in tests?

Reply:
unlike usual deprecations, this one tells `For almost all use cases...`. Tests were and are happy with math rand. I just satisfied the linter

afaik we're doin the same in other repos, right? @roman-khimov

Reply:
Yes. Technically `math/rand` is faster and it works fine for tests. We may reconsider this in https://github.com/nspcc-dev/neofs-sdk-go/issues/552 where the whole `math/rand` gets replaced with v2.